src/Dispatcher.h

```diff
@@ -63,6 +63,9 @@ class Dispatcher {
             }
 
             Nan::MakeCallback(Nan::GetCurrentContext()->Global(), callback, arguments.size(), &arguments[0]);
+
+            delete event->message;
```

Comment:
Message isn't always guaranteed to be populated (in the case of onCreate, onLogon, onLogout callbacks), so calling delete here will result in undefined behavior.